js/headers.ts

```diff
@@ -79,7 +79,8 @@ class HeadersBase {
 
 // @internal
 // tslint:disable-next-line:variable-name
-export const Headers = DomIterableMixin<string, string, typeof HeadersBase>(
-  HeadersBase,
-  headerMap
-);
+export class Headers extends DomIterableMixin<
+  string,
+  string,
+  typeof HeadersBase
+>(HeadersBase, headerMap) {}
```

Comment:
@kitsonk Is this ok with you? I was getting type errors that were resolved when I did this.

Reply:
Wow, I honestly didn't realise that worked, so no, no problem at all. 👍